build-release-apk.sh

```
@@ -0,0 +1,27 @@
+OUTPUT_DIR="${1:-$(pwd)/release-apk}"
+IMAGE_TAG="musly-release-builder"
+CONTAINER_NAME="musly-apk-build"
+
+cd "$(dirname "$0")"
+
+echo "==> Building Docker image: $IMAGE_TAG"
+sudo docker build -f Dockerfile.release -t "$IMAGE_TAG" .
+
+echo "==> Extracting APK from image"
+sudo docker rm -f "$CONTAINER_NAME" 2>/dev/null || true
+sudo docker create --name "$CONTAINER_NAME" "$IMAGE_TAG" echo done
```

Shellcheck (0.11.0)

[warning] Use semicolon or linefeed before 'done' (or quote to make it literal).

(SC1010)